cli/popper/utils.py

```diff
@@ -96,21 +96,44 @@ def is_popperized():
     return os.path.isfile(config_filename)
 
 
-def update_config(name, stages, envs, relative_path):
-    """Updates the configuration for a pipeline"""
-
-    if name == 'paper':
-        stages = 'build'
+def update_config(name, stages='', envs='', vars=[], relative_path=''):
+    """Updates the configuration for a pipeline."""
 
     config = read_config()
+    try:
+        pipeline_config = config['pipelines'][name]
+
+        if name == 'paper':
```

Comment:
a `paper` pipeline is the same as any other pipeline, there is no need to create a special case for it